tests/Block/AdminPreviewBlockServiceTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Tests\Block;
+
+use Sonata\AdminBundle\Admin\Pool;
+use Sonata\AdminBundle\Block\AdminPreviewBlockService;
+use Sonata\BlockBundle\Test\AbstractBlockServiceTestCase;
+
+/**
+ * @author Javier Spagnoletti <phansys@gmail.com>
+ */
+class AdminPreviewBlockServiceTest extends AbstractBlockServiceTestCase
```

Comment:
Good first start, can you please add tests for the execute method too

Reply:
Done.